src/coreclr/debug/daccess/enummem.cpp

```diff
@@ -26,6 +26,20 @@
 #include <interoplibabi.h>
 #endif // FEATURE_COMWRAPPERS
 
+#if TARGET_UNIX
```

Comment:
The stowed exception logic isn't something we likely need to support initially. I would update the `#ifdef` with an `AND !defined(TARGET_UNIX)`

Reply:
Ok we have to change also dacimpl.h that conditionally has EnumMemStowedException